Source/AIExtensions/Private/StateTree/Tasks/AIExtStateTreeTaskGetDistanceBetweenActors.cpp

```diff
@@ -10,7 +10,11 @@ EStateTreeRunStatus FAIExtStateTreeTaskGetDistanceBetweenActors::Tick( FStateTre
 
     auto & instance_data = context.GetInstanceData( *this );
 
-    if ( instance_data.Actor1 != nullptr && instance_data.Actor2 != nullptr )
+    if ( instance_data.Actor1 != nullptr && instance_data.OtherComponent != nullptr )
```

Comment:
I would check 

```
if ( instance_data.Actor1 == nullptr )
{
    return EStateTreeRunStatus::Failed;
}

if ( instance_data.OtherComponent != nullptr )
{
    // ...
}
else if ( instance_data.Actor2 != nullptr )
{
    // ...
}
else
{
    return EStateTreeRunStatus::Failed;
}
```

Reply:
Can Context parameters be nullptr ?
State Tree doesn't compile if parameter is not Bound.

Reply:
even if their own tasks in the engine they check for that so I guess it's a good habit :)